### PR TITLE
fix(machine-metrics): perform daemon-reload for upgrades to 1.0.19

### PR DESCRIPTION
**Pull request by nathan-paperspace**
Upgrading an installed PS Agent to 1.0.19 requires a daemon reload because the unit file changes.

Always performing a systemd daemon reload is something of a large hammer, but it should be harmless so I went this route as the quickest solution.

### CODE REVIEW

File: roles/ps_agent/tasks/main.yml [hunk 1]
```diff
@@ -1,43 +1,39 @@
 - name: Install gpg
-  become: yes
```
Comment (colin-welch) [inline roles/ps_agent/tasks/main.yml:2]: Dropping root seems like outside the scope of this pr

File: roles/node_exporter/templates/node-exporter.service [hunk 2]
```diff
@@ -6,7 +6,7 @@ After=network-online.target
 [Service]
 User=root
 Type=simple
-ExecStart={{ node_exporter_path }} --collector.textfile.directory /var/node_exporter/textfiles --web.listen-address={{ node_exporter_bind_addr }}
```
Comment (colin-welch) [inline roles/node_exporter/templates/node-exporter.service:9]: Is this safe to drop?

File: roles/ps_agent/tasks/main.yml (hunk 1 above)
Comment (nathan-paperspace) [reply roles/ps_agent/tasks/main.yml:2]: So the playbook has `become: true` (https://github.com/Paperspace/machine-metrics/blob/main/metrics.yml#L2). All these are redundant in this context, since this role is not intended to be called separately from the playbook.

File: roles/node_exporter/templates/node-exporter.service (hunk 2 above)
Comment (nathan-paperspace) [reply roles/node_exporter/templates/node-exporter.service:9]: In this context the textfile collector isn't being used, so I'm eliminating it.

However, the web listen address is missing port 9100. I've remedied it in a commit.

I have verified it's working on a test box.

**Review by colin-welch — APPROVED** [review]
